azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -99,59 +107,82 @@ def test_auth_provider_creation(self, mocker, client_class, connection_string, c
     @pytest.mark.parametrize(
         "ca_cert",
         [
-            pytest.param(None, id="No CA certificate"),
+            pytest.param(None, id=" No CA certificate "),
             pytest.param("some-certificate", id="With CA certificate"),
         ],
     )
+    @pytest.mark.parametrize(
+        "websockets",
+        [pytest.param(None, id=" No Websockets "), pytest.param(True, id=" With Websockets ")],
+    )
     def test_pipeline_creation(
-        self, mocker, client_class, connection_string, ca_cert, mock_pipeline_init
+        self, mocker, client_class, connection_string, ca_cert, websockets, mock_pipeline_init
     ):
         mock_auth = mocker.patch(
             "azure.iot.device.iothub.auth.SymmetricKeyAuthenticationProvider"
         ).parse.return_value
 
+        mock_config = mocker.patch(
```

Comment:
Here and everywhere - Need to have a test showing that the config is set up correctly, i.e. is called with the correct args

Reply:
done.